.github/workflows/news_update.yml

```diff
@@ -55,14 +55,15 @@ jobs:
 
     # Run the news scraper
     - name: Scrape News
+      id: scrape_news
+      # The scraper may error on some counties but succeed on others. We want to
+      # continue so that we still generate PRs for partial successes. We'll handle
+      # failures in a later step.
+      continue-on-error: true
       run: |
         echo "::set-env name=SCRAPER_TIME::$(date)"
         cd ${GITHUB_WORKSPACE}/scraper
-        # TODO: stop explicitly listing every county when the scraper repo is
-        # updated to scrape every county by default. (We have to do this first
-        # before the scraper can update, otherwise it would break older code in
-        # this repo!)
-        python scraper_news.py alameda contra_costa marin napa san_francisco san_mateo santa_clara solano sonoma --format json_feed --format rss --output "${GITHUB_WORKSPACE}/site/data/news/"
+        python scraper_news.py sonoma --format json_feed --format rss --output "${GITHUB_WORKSPACE}/site/data/news/" 2> errors.txt
```

Comment:
Only sonoma?

Reply:
Whoops, this was supposed to remove all of them. Should be fixed now!